_security/access-control/index.md

```diff
@@ -5,7 +5,7 @@ nav_order: 75
 has_children: true
 has_toc: false
 redirect_from:
-  - /security/access-control/
+  - /security-plugin/access-control/index/
```

Comment:
Although unrelated to this PR, can we create an issue to use variables for common strings like `security` to make changes in future easier. Not sure if this is already in the backlog.

Reply:
It's something we've considered before. Feel free to propose it as an issue.